### PR TITLE
ENH: Parse schema name in sql uri

### PR DESCRIPTION
**Pull request by llllllllll**
I am not sure about the best practices of testing, or if this code live in the most correct spot.

### CODE REVIEW

**Comment by llllllllll** [comment]
status?

File: odo/backends/sql.py [hunk 1]
```diff
@@ -431,7 +439,15 @@ def resource_sql(uri, *args, **kwargs):
     # we were also given a table name
     if args and isinstance(args[0], str):
         table_name, args = args[0], args[1:]
+        try:
+            schema_name, table_name = table_name.split('.', 1)
+        except:
```
Comment (cpcloud) [inline odo/backends/sql.py]: Can you catch only the exception that raised here?

Comment (llllllllll) [inline odo/backends/sql.py]: sorry about that, sure thing.

**Comment by cpcloud** [comment]
I think we should dispatch on the dialect here. I'll show an example tomorrow.

**Comment by llllllllll** [comment]
kk

**Comment by llllllllll** [comment]
Closing in favor of #223

**Pull request closed without merging**